### PR TITLE
profiles: replace x11 socket blacklist with disable-X11.inc

### PR DESCRIPTION
**Pull request by kmk3**
Replace all occurrences of `blacklist /tmp/.X11-unix` with
`include disable-X11.inc`, which blacklists more X11-related files.

Commands used to search and replace:

    $ git grep -Ilz '^blacklist /tmp/.X11-unix' -- \
      etc/profile*/*.profile | xargs -0 perl -0 -pi -e '\
        s/\nblacklist \/tmp\/.X11-unix\n/\n/; \
        s/(\ninclude disable-xdg.inc\n)/\ninclude disable-X11.inc$1/; \
        s/(\ninclude disable-[^Xx\n]+\n)(\n|# )/$1include disable-X11.inc\n$2/'

Note: The following files were also edited manually:

* etc/profile-a-l/erd.profile
* etc/profile-a-l/links-common.profile
* etc/profile-m-z/termshark.profile
* etc/profile-m-z/tmux.profile
* etc/profile-m-z/tshark.profile

Relates to #4462 #4854 #5544.

### CODE REVIEW

**Review by rusty-snake — APPROVED** [review]

**Comment by kmk3** [comment]
Note: I did not manage to call `include disable-X11.inc` from the code itself
without errors, so I left just the main profile search/replace for this PR.

**Review by glitsj16 — APPROVED** [review]

File: etc/profile-a-l/gconf-editor.profile [hunk 1]
```diff
@@ -7,13 +7,13 @@ include gconf-editor.local
 # added by included profile
 #include globals.local
 
-blacklist /tmp/.X11-unix
```
Comment (glitsj16) [inline etc/profile-a-l/gconf-editor.profile:10]: This never should have been in the profile before this PR, my mistake originally. It's a GUI app and it obviously breaks when including disable-X11.inc.

Not sure how best to proceed to fix it.

**Review by glitsj16 — COMMENTED** [review]
See gconf-editor.profile comment.

File: etc/profile-a-l/gconf-editor.profile (hunk 1 above)
Comment (kmk3) [reply etc/profile-a-l/gconf-editor.profile:10]: > This never should have been in the profile before this PR, my mistake
> originally. It's a GUI app and it obviously breaks when including
> disable-X11.inc.
> 
> Not sure how best to proceed to fix it.

I'll remove it after this PR.

Comment (glitsj16) [reply etc/profile-a-l/gconf-editor.profile:10]: Thanks!

Comment (kmk3) [reply etc/profile-a-l/gconf-editor.profile]: > I'll remove it after this PR.

Fixed it on master and rebased the PR to make the result a bit less confusing.

Anyway, good catch.

**Pull request merged**